flink-streaming-java/src/test/java/org/apache/flink/streaming/api/graph/SinkTransformationTranslatorITCase.java

```diff
@@ -225,7 +225,8 @@ private StreamNode findWriter(StreamGraph streamGraph) {
     }
 
     private StreamNode findCommitter(StreamGraph streamGraph) {
-        return findNodeName(streamGraph, name -> name.contains("Committer"));
+        return findNodeName(
```

Comment:
Can you explain why we need this change?

Reply:
> Can you explain why we need this change?

This test is unstable when ITCase runs in IDEA that `Global Committer` node could be wrongly found as `Committer`. This change may solve it to avoid risk. Do you think it is necessary?